src/cli.rs

```diff
@@ -335,6 +335,28 @@ pub(crate) enum Command {
         cmd: CollapseReadsToFragmentsSubcommand,
     },
 
+    /// Tool to build artifical reads from real BAM files with identical properties.
+    #[structopt(author = "Felix Mölder <felix.moelder@uni-due.de>")]
+    SimulateReads {
+        #[structopt(parse(from_os_str), help = "Input BAM file")]
+        bam: PathBuf,
+        #[structopt(parse(from_os_str), help = "Input reference as fasta file")]
+        input_ref: PathBuf,
+        #[structopt(parse(from_os_str), help = "Output BAM file with artificial reads")]
+        output_bam: PathBuf,
+        #[structopt(
+            parse(from_os_str),
+            help = "Output fasta file with artificial reference"
+        )]
+        output_ref: PathBuf,
+        #[structopt(help = "chromosome name")]
+        chr: String,
+        #[structopt(help = "0-based start position")]
+        start: u64,
+        #[structopt(help = "0-based exclusive end position")]
+        end: u64,
```

Comment:
Use 1-based positions here to be in line with samtools interface.

Reply:
It turned out that I already used 1-based coordinates internally. So I only had to double check and fix the description.